### DISCUSSION TITLE
Welcome to omedis Discussions!

### DISCUSSION THREAD

**Discussion by wintermeyer** [3Stones-io/omedis#220] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fradarr** [comment]
Hi Stefan please lead by example and introduce yourself first : ) 

Me I'm Denis and I'm the project lead. 45, 5 Kids, living and working out of Luzern/Switzerland. Looking into learnig ASH and Elixir! 
Looking forward working with you guys.

**Comment by wintermeyer** [comment]
Truth be told: The original posting was a canned one by GitHub. I didn't even read it. 😉

About me: I am 51 years old. 2 teenaged kids. One crazy dog. German. Living in Koblenz (am Rhein). IT dinosaur.

**Reply by almirsarajcic** [reply]
Yeah, I thought the same. That's why I ignored the thread initially.

**Reply by fradarr** [reply]
Oups, well nice hearing some more stuff about you anyway... So Stefan has got an crazy german shepherd : ) 
"...One crazy dog. German..."

**Reply by wintermeyer** [reply]
> crazy german shepherd : )

Not a German Shepherd. A "English Springer Spaniel". In total I had 4 dogs before but this one is the most challenging. My advise: "Never buy a Springer Spaniel! Never!" 😂

**Reply by fradarr** [reply]
I can imagine they look like chase, chase, chase 

<img width="673" alt="Screenshot 2024-10-29 at 16 47 57" src="https://github.com/user-attachments/assets/550e5b3e-577a-4056-9be0-38c908827c4b">


I had last Jack Russel (x2) (lovely but crazy too) 
and Staff x Boxer before (beautifull kind dog)

**Reply by wintermeyer** [reply]
> I can imagine they look like chase, chase, chase

They are. But the good thing about this breed is that they keep a short distance all the time. My former dog was an English Setter. Super nice and super friendly. The perfect family dog. But always at the horizon never near by. The English Springer Spaniel is always in a 10 meter radius around me. So much easier to walk in the woods. And it is a good family dog too. Just a very active one. 😉

**Comment by almirsarajcic** [comment]
32 years old Bosnian living in Sarajevo
I currently don't live with my wife and kids as they live in the future. In other words, I'm all alone 😭😭😭

**Reply by fradarr** [reply]
Can't force anything in live, but that beeing said... go chasing bro : )

**Comment by amos-kibet** [comment]
Former electrician living in a remote town (Bomet) in Kenya. I'm working as a fullstack developer on this project.

**Reply by fradarr** [reply]
<img width="717" alt="Screenshot 2024-10-29 at 16 43 40" src="https://github.com/user-attachments/assets/5cd53b4d-bf30-4431-9ce8-195f23e99a1f">
So interesting names of places : ) Nice to have you on the project!

**Reply by wintermeyer** [reply]
> Former electrician

Is it true that EVERY electrician has at least one or two high voltage "experiences" in this professional live? I got hit with 220 V twice and THAT is something I'll never forget.

**Reply by almirsarajcic** [reply]
Seems interesting. You get immediate feedback when you make mistakes.
That gives me an idea: maybe we should get electrocuted whenever we create a bug.
Maybe I shouldn't manage software developers (or anyone for that matter) 😆

**Reply by amos-kibet** [reply]
> Is it true that EVERY electrician has at least one or two high voltage "experiences" in this professional live? I got hit with 220 > V twice and THAT is something I'll never forget.

I have three such experiences. In one of them, my hand got paralyzed for like 2 seconds.

> Seems interesting. You get immediate feedback when you make mistakes.
> That gives me an idea: maybe we should get electrocuted whenever we create a bug.
> Maybe I shouldn't manage software developers (or anyone for that matter) 😆

Feedback is immediate, and brutal!

**Reply by wintermeyer** [reply]
> That gives me an idea: maybe we should get electrocuted whenever we create a bug. 

https://youtu.be/vUc4GkMN1qs?si=rhzdCTn5XzXzTlgA&t=129

**Comment by kagure-nyakio** [comment]
I am mostly a nomad, but I spend most of my days in Nanyuki with my niece. I am working as a full-stack developer on this project.
